### DISCUSSION TITLE
failed to find font 'DejaVu Serif, serif'

### DISCUSSION THREAD

**Discussion by undarbiko** [tdewolff/canvas#258] [Q&A]
Hello,
I am new to golang, few days I am playing around. 
When I try to run opengl or fyne example I am getting panic error:

```
$ go run -tags latex main.go
panic: failed to find font 'DejaVu Serif, serif'

goroutine 1 [running, locked to thread]:
main.main()
        D:/sources/go/tdewolff_canvas/opengl/main.go:21 +0x406
exit status 2
``` 

I am working with Windows 10 with Msys2 installed. The DejaVu Serif fonts are present in windows and I also installed the mingw-w64-x86_64-ttf-dejavu and mingw-w64-i686-ttf-dejavu packages for Msys. 
Any clues what is wrong with my setup?
Thank you!

**Comment by tdewolff** [comment]
This must be a problem on Windows since it works fine on Linux and unfortunately I can't test on Windows.

Could you help me debug? What is the output of the following code for you?

```go
package main

import "github.com/tdewolff/canvas/font"

func main() {
    fmt.Printf("SYSTEMROOT '%s'\n", os.Getenv("SYSTEMROOT"))
    fmt.Printf("SYSTEMDRIVE '%s'\n", os.Getenv("SYSTEMDRIVE"))
    dirs := font.DefaultFontDirs()
    fmt.Println("DefaultFontDirs", dirs)
    fonts, err := font.FindSystemFonts(dirs)
    fmt.Println("FindSystemFonts:", fonts, err)
    font, ok := fonts.Match("DejaVu Serif, serif", font.Regular)
    fmt.Println("Match", font, ok)
}
```
